src/objects.js

```diff
@@ -1,43 +1,59 @@
 const createPerson = (name, age) => {
-  // your code here
-};
+  return {
+    'name': name,
+    'age': age
+  }
+}
 
 const getName = object => {
-  // your code here
+  return object.name; 
 };
 
 const getProperty = (property, object) => {
-  // your code here
+  return object[property];
 };
 
 const hasProperty = (property, object) => {
-  // your code here
+  return object.hasOwnProperty(property);
 };
 
 const isOver65 = person => {
-  // your code here
+ return person.age > 65 ? true : false; 
```

Comment:
As above, you could simply return the result of evaluating the condition since you are only looking for a boolean value here:
```
  return person.age > 65;
```